cmake/fficonfig.h.in

```diff
@@ -28,14 +28,14 @@
 /* #undef FFI_NO_STRUCTS */
 
 /* Define to 1 if you have `alloca', as a function or macro. */
-#define HAVE_ALLOCA 1
+#cmakedefine HAVE_ALLOCA 1
 
 /* Define to 1 if you have <alloca.h> and it should be used (not on Ultrix).
    */
-#define HAVE_ALLOCA_H 1
+#cmakedefine HAVE_ALLOCA_H 1
 
 /* Define if your assembler supports .cfi_* directives. */
-#define HAVE_AS_CFI_PSEUDO_OP 1
+#cmakedefine HAVE_AS_CFI_PSEUDO_OP 1
```

Comment:
this flag is only being used on m68k, it won't matter in other places.

introspection via ``asm(".cfi_startproc\n\t.cfi_endproc");`` may still be architecture / assembly dependent. Post on gcc mailing list https://gcc.gnu.org/ml/gcc-patches/2014-04/msg00310.html.

I can keep the current behavior and make a note about it.  notes: https://gist.github.com/tony/08c2cbf23eb80a7737ea9e2829992ca7

I don't have a way to test and can't sink more time into it atm.